web_src/js/features/common-issue.js

```diff
@@ -2,15 +2,34 @@ import $ from 'jquery';
 import {updateIssuesMeta} from './repo-issue.js';
 
 export function initCommonIssue() {
-  $('.issue-checkbox').on('click', () => {
-    const numChecked = $('.issue-checkbox').children('input:checked').length;
-    if (numChecked > 0) {
+  let checkboxfirst;
+  const checkboxOperate = (e) => {
+    const issuecheckbox = $('.issue-checkbox input');
+    if (e.shiftKey && checkboxfirst !== undefined) {
```

Comment:
This currently only works when the checkbox is below the previous checkbox. Which can be confusing behavior that the other way around doesn't work, also might be something to do for another PR to minimize the changes in this PR.

Reply:
Thanks, ok, i deleted it